Python/Tests/Core.UI/VirtualEnvUITests.cs

```diff
@@ -54,52 +50,28 @@ public class VirtualEnvTests {
             }
         }
 
-        static DefaultInterpreterSetter Init(PythonVisualStudioApp app, PythonVersion interp, bool install) {
-            var dis = app.SelectDefaultInterpreter(interp);
-            try {
-                if (install) {
-                    dis.CurrentDefault.PipInstall("-U virtualenv");
-                }
-                var r = dis;
-                dis = null;
-                return r;
-            } finally {
-                dis?.Dispose();
-            }
-        }
-
-        static DefaultInterpreterSetter Init3(PythonVisualStudioApp app, bool installVirtualEnv = false) {
-            var dis = app.SelectDefaultInterpreter(
+        static DefaultInterpreterSetter InitPython3(PythonVisualStudioApp app) {
+            return app.SelectDefaultInterpreter(
+                PythonPaths.Python36 ?? PythonPaths.Python36_x64 ??
```

Comment:
Add Python37 here as well. When the test machines come back up I'll put 3.7.0a2 on them.